x11-libs/wxGTK/wxGTK-3.0.4-r3.ebuild

```diff
@@ -94,6 +94,11 @@ multilib_src_configure() {
 	fi
 
 	ECONF_SOURCE="${S}" econf "${myconf[@]}"
+		# PCHes are unstable and are disabled in-tree where possible
+		# See bug #504204
+		# Commits 8c4774042b7fdfb08e525d8af4b7912f26a2fdce, fb809aeadee57ffa24591e60cfb41aecd4823090
+		$(use_enable pch precomp-headers)
```

Comment:
Wrong place

Reply:
Whoops, misapplied my own patch

Fixed in ed9c73deaedebd147462b51bab58e858277bfea0